### ISSUE TITLE
Add a automatic invert function

### ISSUE THREAD

**Issue by sun-geo** [opening-hours/opening_hours.js#352]
Lets say we have a opening/service time of 
`Mo-Fr 09:00-17:00`
and for conditinal syntax you want to write the inverted times like:
`Mo-Fr 00:00-09:00,17:00-24:00, Sa,Su 00:00-24:00`

**Comment by ypid** [comment]
Hm, I don’t have a clear idea yet how I could program this so that the inverse that would be generated is actually an "optimal" way to express it. So before we talk about extending https://openingh.openstreetmap.de/evaluation_tool/, lets maybe talk about how the syntax can be used in clever yet defined ways to achieve what you want.

My idea:

```
open; Mo-Fr 09:00-17:00 off
Mo-Fr 09:00-17:00
```

See https://wiki.openstreetmap.org/wiki/Key:opening_hours/specification#explain:rule_modifier:closed

This makes it really quick to "toggle" inverse and normal. But that is open for discussion. It was just the first thing I came up with right now.

**Comment by sun-geo** [comment]
> My idea:
> 
> ```
> open; Mo-Fr 09:00-17:00 off
> Mo-Fr 09:00-17:00
> ```

Interesting, the request was related in usage of date/time values for traffic signs. I never used the `open` / `closed` / `off` syntax in combination with traffic signs. Do you think it would be okay? Or you want to use the syntax just for an internal converting function?

**Comment by ypid** [comment]
Yes, it is already supported (for years already). I guess now that I read it a second time, I don’t see anything wrong with this so I would say you can use it.
